src/Microsoft.AspNet.Server.Kestrel/Infrastructure/MemoryPoolIterator2.cs

```diff
@@ -228,18 +228,13 @@ public int Seek(int char0)
             }
         }
 
-        public int Seek(int char0, int char1)
+        public int Seek(Vector<byte> byte0Vector, Vector<byte> byte1Vector)
```

Comment:
No need to change it. @cesarbs has already reviewed it to make sure you didn't screw anything up with the renames :stuck_out_tongue: